sos/plugins/__init__.py

```diff
@@ -323,7 +323,12 @@ def do_file_sub(self, srcpath, regexp, subst):
                 replacements = 0
         except Exception as e:
             msg = "regex substitution failed for '%s' with: '%s'"
-            self._log_error(msg % (path, e))
+            # if trying to regexp a nonexisting file, dont log it as an
+            # error to stdout
+            if e.errno == errno.ENOENT:
```

Comment:
I'd prefer the comment to say something like "Processing a file that was not collected is not an error" - I am not even sure we need an info log in that event. We already know it wasn't collected.

Reply:
OK, verbosity decreased to debug and log statement rephrased.